pkg/serverless/tags/tags.go

```diff
@@ -183,9 +191,11 @@ func setIfNotEmpty(tagMap map[string]string, key string, value string) map[strin
 }
 
 func addTag(tagMap map[string]string, tag string) map[string]string {
-	extract := strings.Split(tag, ":")
+	extract := strings.SplitN(tag, ":", 2)
 	if len(extract) == 2 {
 		tagMap[strings.ToLower(extract[0])] = strings.ToLower(extract[1])
+	} else {
+		log.Debug("Tag" + tag + " has not expected format")
```

Comment:
According to our documentation, tags without colon are allowed:

> A tag can be in the format `value` or `<KEY>:<VALUE>`

Why `value` format is not allowed here?